projects/plugins/search/tests/e2e/helpers/search-helper.js

```diff
@@ -69,18 +105,20 @@ export async function searchAPIRoute( page ) {
 
 		// deal with sorting
 		switch ( params.get( 'sort' ) ) {
-			case 'date_asc':
+			case 'date_asc': {
```

Comment:
TIL this was a thing...

Reply:
In case you didn't find it already, https://eslint.org/docs/latest/rules/no-case-declarations explains why this change.